packages/roosterjs-editor-core/lib/coreApi/addUndoSnapshot.ts

```diff
@@ -81,3 +85,73 @@ export const addUndoSnapshot: AddUndoSnapshot = (
         }
     }
 };
+
+// interface SnapshotSelectionBase<T extends SelectionRangeTypes> {
```

Comment:
There is some commented code here, should we keep it?

Reply:
oops, forget to remove